ale_linters/kotlin/kotlinc.vim

```diff
@@ -23,6 +24,20 @@ function! ale_linters#kotlin#kotlinc#GetImportPaths(buffer) abort
                         \ . 'mvn dependency:build-classpath'
         endif
 
+        let l:build_path = ale#path#FindNearestFile(a:buffer, 'build.gradle')
+
+        if !empty(l:build_path)
+            let l:gradlew_path = ale#path#FindNearestFile(a:buffer, 'gradlew')
+            
+            if !empty(l:gradlew_path)
+                return ale#path#CdString(fnamemodify(l:gradlew_path, ':h'))
+                            \ . './gradlew -I ' . s:script_path . '/init.gradle -q printClasspath'
```

Comment:
Write your line continuations like so.

```vim
foo
\   . bar
```

You can look through the codebase for examples.